plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/PluginModernizer.java

```diff
@@ -70,9 +73,10 @@ private void process(Plugin plugin) {
                 plugin.removeLocalData();
             }
             plugin.fetch(ghService);
-            plugin.clean(mavenInvoker);
+            plugin.compile(mavenInvoker, jdkFetcher.getJdkPath("8"));
```

Comment:
Will cause issue on plugin that will require a core version or dependencies requiring modern JDK (or even 11)

Reply:
Yep! Need to figure a way to choose JDK automatically (based on plugins) instead of hard-coding it.